File: Selenium/Activities/activity_6.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:

    driver.get("https://training-support.net/webelements/dynamic-controls")

    print("Title of the page: ", driver.title)

    checkbox = driver.find_element(By.CSS_SELECTOR, "#checkbox")

    print("Is checkbox selected? ", checkbox.is_selected())

    checkbox.click()

    print("Is checkbox selected after clicking? ", checkbox.is_selected())

    driver.quit()

File: Selenium/Activities/Activity6.java
package activity;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity6 {

	public static void main(String[] args) {
		
		WebDriver driver = new ChromeDriver();
		
		driver.get("https://training-support.net/webelements/dynamic-controls");
		
		System.out.println("Title of the page: " + driver.getTitle());
		
		WebElement checkbox = driver.findElement(By.cssSelector("#checkbox"));
		
		System.out.println("Is checkbox selected? "+ checkbox.isSelected());
		
		checkbox.click();
		
		System.out.println("Is checkbox selected after clicking? "+ checkbox.isSelected());
		
		driver.quit();

	}

}
